src/registrar/context_processors.py

```diff
@@ -70,16 +66,22 @@ def portfolio_permissions(request):
                 "has_base_portfolio_permission": False,
                 "has_domains_portfolio_permission": False,
                 "has_domain_requests_portfolio_permission": False,
+                "portfolio": None,
```

Comment:
portfolio context removed from individual views get_context_data since context effects many views across the app and therefore is added globally in context_processors